bin/build_cmd.ml

```diff
@@ -217,6 +217,12 @@ let build =
          perform the RPC call.
       *)
       Scheduler.go_without_rpc_server ~common ~config (fun () ->
+        User_warning.emit
+          [ Dune_util.Global_lock.Lock_held_by.message lock_held_by
+          ; Pp.text
+              "The requested targets will still be built however functionality will be \
+               reduced and some command line arguments will be ignored."
+          ];
```

Comment:
Would it be possible to adjust this error message to only display the warning when there are arguments that are being ignored? This way it warns about arguments being ignored even if there are no arguments.